source/API_Reference/Event_Webhook/event.md

```diff
@@ -0,0 +1,1043 @@
+- <a name="sgeventid"></a>`sg_event_id` - A unique ID to this event that you can use for deduplication purposes. These IDs are either 22 or 48 characters.
+- <a name="sgmessageid"></a>`sg_message_id` - An internal SendGrid unique ID for the message type. The first half of this is pulled from the `smtp-id`.
+- <a name="reason"></a>`reason` - Any sort of error response returned by the server that describes the reason this event type was triggered.
+- <a name="status"></a>`status` - Status code string. Corresponds to HTTP status code - for example, a JSON response of 5.0.0 is the same as a 500 error response.
+- <a name="response"></a>`response` - The full text of the HTTP response error returned from the server.
+- <a name="tls"></a>`tls` - Indicates whether TLS encription was used in sending this message. For more information about TLS, see the [TLS Glossary page]({{root_url}}/Glossary/tls.html).
+- <a name="url"></a>`url` - The URL where the event originates, unless it's a click event, and this is the URL clicked by the reciepient.
+- <a name="attempt"></a>`attempt` - The number of times the server has attempted to deliver this mail.
+- <a name="category"></a>`category` - [Categories]({{root_url}}/Glossary/categories.html) are custom tags that you set for the purpose of organizing your emails. If you send single categories as an array, they will be returned by the webhook as an array. If you send single categories as a string, they will be returned by the webhook as a string.
+
+String categories:
+
+{% codeblock lang:json %}
+[
+  {
+    "email": "john.doe@sendgrid.com",
+    "timestamp": 1337966815,
+    "category": "newuser",
+    "event": "open"
+  },
+  {
+    "email": "jane.doe@sendgrid.com",
+    "timestamp": 1337966815,
+    "category": "olduser",
+    "event": "open"
+  }
+]
+{% endcodeblock %}
+
+Array:
+
+{% codeblock lang:json %}
+[
+  {
+    "email": "john.doe@sendgrid.com",
+    "timestamp": 1337966815,
+    "category": [
+      "newuser",
+      "transactional"
+    ],
+    "event": "open"
+  },
+  {
+    "email": "jane.doe@sendgrid.com",
+    "timestamp": 1337966815,
+    "category": "olduser",
+    "event": "open"
+  }
+]
+{% endcodeblock %}
+
+- <a name="asmgroupid"></a>`asm_group_id` - The ID of the unsubscribe group the recipients email is included in. ASM IDs correspond to the ID that is returned when you create an unsubscribe group.
+- <a name="uniqueargs"></a>`unique_args` or `custom_args` - 
+{% anchor h2 %}
+Unique Arguments and Custom Arguments
+{% endanchor %}
+
+Events generated by SendGrid may also include any [unique arguments]({{root_url}}/API_Reference/SMTP_API/unique_arguments.html) or custom arguments.
+
+{% info %}
+Unique arguments and custom arguments are functionally the same: It's called unique arguments in the SMTP API or v2 Mail Send, and it's called custom arguments in v3 Mail Send.
```

Comment:
In the SMTP API or V2 Mail Send, they're referred to as unique arguments and are known as custom arguments in V3 Mail Send.